task/07-yield-tasks.js

```diff
@@ -126,7 +156,22 @@ function* breadthTraversalTree(root) {
  *   [ 1, 3, 5, ... ], [ -1 ] => [ -1, 1, 3, 5, ...]
  */
 function* mergeSortedSequences(source1, source2) {
-    throw new Error('Not implemented');
+    const first = source1();
+    const second = source2();
+
+    while (true) {
+        let a = first.next().value;
+        let b = second.next().value;
+
+        if (a == undefined)
+            yield b;
+        else if (b == undefined)
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"